src/main/webapp/script.js

```diff
@@ -12,7 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function openFamilyForm() {
+    document.getElementById("createFamilyForm").style.display = "block";
```

Comment:
You can use visibility: https://www.w3schools.com/jsref/prop_style_visibility.asp